### ISSUE TITLE
Shader for point cloud rendering ?

### ISSUE THREAD

**Issue by huohuohuohuohuohuohuohuo** [facebookresearch/pytorch3d#1578]
## ❓ Questions on how to use PyTorch3D

Thanks for the amazing work! In Section 3.3 Differentiable point cloud renderer in the offical paper, Accelerating 3D Deep Learning with PyTorch3D, it mentions that pytorch3d provides shaders for  silhouette and textured point cloud rendering, and users can easily implement custom shaders to customize the rendering pipeline. 

However, In the Class PointsRenderer(nn.Module), its member variables only contain rasterizer and compositor. May I ask how to use shaders considering lighting or others for point cloud rendering ?

**Comment by bottler** [comment]
You are right that the "shading" functions and lighting capabilities in pytorch3d are for meshes and not for point clouds. It is possible for a user to implement their own compositor to make the PointsRenderer do what they want.
